modules/markets-index/src/main/scala/org/ergoplatform/dex/index/processes/PoolsIndexing.scala

```diff
@@ -62,12 +67,21 @@ object PoolsIndexing {
         val assets        = poolSnapshots.flatMap(p => List(p.entity.lp.id, p.entity.x.id, p.entity.y.id)).distinct
         val commit        = batch.lastOption.map(_.commit)
 
+        val retryPolicy = limitRetries[F](30) |+| constantDelay[F](10.seconds)
```

Comment:
Point to discussion @oskin1:

In all sql queries we think, that token info exists for pool in db. But the situation when we failed to get token info in the end of our retry policy is possible. 

We have some ways to resolve this issue:
1. Simply drop pool for token without info. We lose some information but it's an analytics services. 
2. Poll explorer forever (and investigate by hands if the poll will be really infinite).
3. The dangerest - insert pool without token info.

Reply:
Normally token info must be present for any pool in explorer db. Usually the error here is caused by non 2xx/4xx response codes (response timeout usually I guess). In this case retry is effectively a solution. In the case of 4xx code we have an inconsistent explorer view and the service should halt indexing and wait for manual investigation.